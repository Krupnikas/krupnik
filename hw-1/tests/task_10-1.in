1 108 3.1415
