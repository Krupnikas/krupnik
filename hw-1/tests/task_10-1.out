Char: l
Int: 1
Real: 3.141500
